src/main.js

```diff
@@ -6,5 +6,48 @@
 // 4. Make CPU smarter? Add difficulty levels (Impossible/Normal/Stupid)?
 
 const { runPlayLoop } = require("../lib/play");
+const { HumanPlayer } = require("../lib/player");
+const { Lobby } = require("../lib/lobby");
 
-runPlayLoop();
+const lobby = new Lobby();
+const player = new HumanPlayer();
+
+runPlayLoop(lobby, player);
```

Comment:
are we planning to make the local version work in this PR or backlog it as a known regression?